CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## 2.8.0 (Unreleased)
+
+BUG FIXES:
+* `resource/vcd_vapp_vm` sometimes reports incorrect `ip` in legacy network configuration [GH-478]
```

Comment:
Please be more specific by "legacy network configuration", for a casual user reading the changelog it raises questions :)

Reply:
Changed

Reply:
Thank you. Another small suggestion: consider changing the word "legacy" to "deprecated" for even greater clarity.